include/irr/asset/ICPUMeshBuffer.h

```diff
@@ -92,6 +97,15 @@ class ICPUMeshBuffer : public IMeshBuffer<ICPUBuffer, ICPUDescriptorSet, ICPURen
     {
         return m_descriptorSet.get();
     }
+	inline void setVertexAttrBuffer(uint16_t attribIndex, E_FORMAT format, uint32_t stride, uint32_t offset)
+	{
+		auto &params(getPipeline()->getVertexInputParams());
```

Comment:
you need to handle attribIndex>=MAX

Reply:
as well as stride>MAX and offset>MAX (there are some maximums that are hard-set)